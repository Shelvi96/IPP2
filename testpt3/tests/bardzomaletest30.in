NEW ID

5>:236
894>0982:3
8648:43>99563
864;08;14>16
84>048
0>93560:580
64>:43
964750;;18>982:475
9>89721;4
6>;106
49:95>764524221
;;3542:>9
529302523:>:85829252
85>1:5346128
@08:701
@210:4111449962;;649;
@953:2692;44
@978:0964:4:172;3;:1
@39;930;688:577
@;8677258:;;1313
@8476;;9579354257;;2:20:
@3956:82649038:88;45
@6:399673653929809:;20;9
@1:81219830:5:58834:
