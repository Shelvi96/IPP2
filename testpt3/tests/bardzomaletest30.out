0
2363392
0
1964568
9
9
26385790706
1021010
10010100100
544572
